addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -138,13 +138,14 @@
             </t>
         </a>
         <ul class="collapse show p-0 m-0 list-unstyled" t-att-id="('collapse-%s') % (category.id if category else 0)" >
+            <t t-set="is_member" t-value="slide.channel_id.is_member"/>
+            <t t-set="can_access_channel" t-value="is_member or slide.channel_id.can_publish"/>
             <t t-foreach="category_slide_ids" t-as="aside_slide">
                 <t t-set="slide_completed" t-value="channel_progress[aside_slide.id].get('completed')"/>
-                <t t-set="is_member" t-value="slide.channel_id.is_member"/>
-                <t t-set="can_access" t-value="aside_slide.is_preview or is_member or slide.channel_id.can_publish"/>
+                <t t-set="can_access_current_slide" t-value="aside_slide.is_preview or can_access_channel"/>
```

Comment:
Is there any change ? Seems it has the same meaning, but not sure ... as in stable we try to have the minimal diff we should avoid changes as much as possible (notably changing variables, as this may break inheriting templates).

Reply:
We can, `can_access` means `can_access_current_slide`... I changed it back. At one spot, we need the `can_access_channel` logic instead however. 

I guess the readability improvement is not worth the extra computation of doing something like this before master:
```xml
<t t-set="can_access_current_slide" t-value="can_access_channel or slide.is_preview"/>
<t t-set="can_access" t-value="can_access_current_slide"/><!-- Deprecated, removed in v15.x -->
```